docs/hostname-change.md

```diff
@@ -1,21 +1,17 @@
 ## Changing the Hostname of a Satellite host
-### **NOTE: This script must be used for Satellite 6.2 only**
-#### On the Satellite host
-1. Before you run the hostname change playbook, you have to make a manual change to the default capsule's hostname. Log into UI and rename the default capsule (under infrastructure->capsules) to the new hostname using both the name and url fields.
+
+### **Important warnings - read before running this playbook**
+ - This script must be used for Satellite 6.2 only
+ - All capsules and satellite clients will have to be re-registered with Red Hat Subscription mananger, and capsules will have to be reinstalled after running this playbook
 
 #### On the Control node
-1. Update the required sections in [roles/change_hostname/vars/main.yml] (roles/change_hostname/vars/main.yml).
-2. Update the [inventory] (inventory) file under the root of this project to add your Satellite host's ip address. If executing the playbook on localhost, add `ansible_connection=local` after the IP address
+1. Create file `roles/change_hostname/vars/main.yml` by copying `roles/change_hostname/vars/main.sample.yml` and update it as necessary.
+2. Update the [inventory] (inventory) file under the root of this project to add your Satellite machine's ip address. If executing the playbook on localhost, add `ansible_connection=local` after the IP address
 3. Run the ansible playbook:
 
     ```console
       # ansible-playbook -i inventory satellite-hostname-playbook.yml
     ```
-4. If you have a capsule you will need to reregister it with RHSM and then run the following (replacing `<capsule-hostname>` with your capsule's hostname):
-
-   ```console
-     # capsule-certs-generate --capsule-fqdn '<capsule-hostname>' --certs-tar '~/<capsule-hostname>-certs.tar'`
-   ```
-Then follow the output generated by the capsule-certs-generate command
+4. Follow the output generated by the katello-change-hostname command. This can be found in change-hostname.log
```

Comment:
Sounds reasonable.  Can you update the following comment then?
`4. Follow the output generated by the katello-change-hostname command. This can be found in change-hostname.log`
This is the statement which made me believe that the playbook will show some progress :)